corehq/apps/users/forms.py

```diff
@@ -1026,16 +1026,19 @@ def __init__(self, *args, **kwargs):
         self.helper.field_class = 'col-sm-9 col-md-8 col-lg-6'
         self.helper.form_tag = False
 
+        from corehq.apps.hqwebapp.utils.bootstrap import get_bootstrap_version, BOOTSTRAP_5
+        bootstrap_version = get_bootstrap_version()
+        use_bootstrap5 = bootstrap_version == BOOTSTRAP_5
+        submit_button_holder = crispy.ButtonHolder(Submit('submit', submit_label))
+        if not use_bootstrap5:
+            submit_button_holder = hqcrispy.FormActions(submit_button_holder)
+
         self.helper.layout = crispy.Layout(
             crispy.Fieldset(
                 fieldset_title,
-                crispy.Field('selected_ids', css_class="hide"),
+                crispy.Field('selected_ids', css_class="d-none" if use_bootstrap5 else "hide"),
```

Comment:
to be commit with above nitpick suggestion if it is accepted
```suggestion
                crispy.Field('selected_ids', css_class="d-none" if is_bootstrap5 else "hide"),
```